### DISCUSSION TITLE
How do I get notified when a (core) formula is updated?

### DISCUSSION THREAD

**Discussion by tddschn** [Homebrew/discussions#2085] [Everyday usage]
`python@3.9` has been updated to `3.9.7` for a while in the upstream, and it's still at version `3.9.6` in the homebrew core tap. I want to be get notified when the formula is updated, how do I do that?

Running `brew update` frequently and subscribing to [this pull request](https://github.com/Homebrew/homebrew-core/pull/84283) isn't the best way to get notified, obviously.

**Comment by carlocab** [comment]
> Running `brew update` frequently and subscribing to [this pull request](https://github.com/Homebrew/homebrew-core/pull/84283) isn't the best way to get notified, obviously.

Why do you say that? It seems to me that these are, in fact, the best ways to get notified.

**Reply by tddschn** [reply]
The pull request contains too much info, and I want to only want to be notified when it's merged. 

Can you subscribe to only the merge event of a pull request on github? I don't think it's implemented yet. :)

**Comment by tddschn** [comment]
Is there something like an RSS feed for a particular core formula?

**Reply by carlocab** [reply]
Not really. There is a JSON API: https://formulae.brew.sh/docs/api/

You could try periodically querying that, perhaps using `cron` or `launchd`.

Some other alternatives:

 1. It might be possible to set up a [Git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) that notifies you when a particular formula is updated.
 2. You can use a file system watcher (e.g. [watchman](https://formulae.brew.sh/formula/watchman#default), [entre](https://formulae.brew.sh/formula/entr#default)) to do something similar.
 3. If you use email to get GitHub notifications, you can use your email filters to filter events on the pull request. If you don't use email, you can just check the icon on the notification in the GitHub UI, because this changes depending on the status of the pull request.

I imagine there's more.

**Reply by tddschn** [reply]
@carlocab These are some really great ideas, thank you!
I want to mark your comment as answer, but GitHub won't allow me to do that. :(
